incubator/hnc/pkg/kubectl/tree.go

```diff
@@ -34,15 +36,24 @@ var (
 var treeCmd = &cobra.Command{
 	Use:   "tree",
 	Short: "Displays the hierarchy tree rooted at the given namespace",
```

Comment:
Can you please add some help text here? E.g., "If no namespace is provided, displays the tree starting at every root namespace on the cluster."